python/sunbeam/deck/parser.py

```diff
@@ -0,0 +1,18 @@
+from os.path import isfile
+import json
+from .. import libsunbeam as lib
+from .. import parser
```

Comment:
the `..` makes me slightly uneasy.  Is there a (nice and clean) way around it?

Reply:
I don't think there is, this is importing from the parent package.